src_new/adapters/prebid.js

```diff
@@ -191,18 +191,17 @@ function getPBCodeWithoutWidthAndHeight(divID, adapterID){
 exports.getPBCodeWithoutWidthAndHeight = getPBCodeWithoutWidthAndHeight;
 /* end-test-block */
 
-function generatedKeyCallback(adapterID, adUnits, adapterConfig, impressionID, generatedKey, kgpConsistsWidthAndHeight, currentSlot, keyConfig, currentWidth, currentHeight){
+function generatedKeyCallback(adapterID, adUnits, adapterConfig, impressionID, generatedKey, kgpConsistsWidthAndHeight, currentSlot, keyConfig, currentWidth, currentHeight, banner_sizes){
```

Comment:
Keep the naming convention of new variables same as existing format, i.e. camel case. Use bannerSizes instead of banner_sizes.

Reply:
Done